XPlat/TensorFlow.Lite/build.cake

```diff
@@ -132,6 +146,47 @@ Task ("clean")
 		}
 	);
 
+Task ("docs-api-diff")
+	.IsDependentOn ("nuget")
+    .Does (async () =>
+{
+    var baseDir = "./output/api-diff";
```

Comment:
I wonder if we should refactor this out to common.cake as well and just make it a simple method call from here with the bare minimum parameters passed in...

Reply:
moved to common.cake and added BuildApiDiff method